docs/source/contributor-guide/roadmap.md

```diff
@@ -46,81 +46,12 @@ make review efficient and avoid surprises.
 
 # Quarterly Roadmap
 
-A quarterly roadmap will be published to give the DataFusion community
-visibility into the priorities of the projects contributors. This roadmap is not
-binding and we would welcome any/all contributions to help keep this list up to
-date.
+The DataFusion roadmap is driven the priorities of the contributors rather than
+any single organization or coordinating committee. We typically discuss roadmap
```

Comment:
should we define the  way of `We typically discuss roadmap` ? weekly meetings, slack, GH, etc?

Reply:
Nvm, I was just thinking it would be more clear to the reader to know where exactly the roadmap is discussed in case they want to participate as well.

Reply:
Thanks -- I tried to make it clearer and explicitly invited the reader to join the discussion